src/Containers/Copyright.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import { SmallText } from '../Text/SmallText';
+
+interface CopyrightProps extends MainInterface, ResponsiveInterface {
+    margin: string;
+    version: string | number;
+}
+
+export const Copyright: React.FC<CopyrightProps> = ({
+    margin,
+    version,
```

Comment:
default this to npm package version

Reply:
Wouldn't this just default to the react-ui package version then?

Reply:
hmm ur right, nvm then :/